### PR TITLE
(SIMP-2553) Update docs for SIMP 6 'simp config'

### PR DESCRIPTION
**Pull request by lnemsick-simp**
SIMP-2553 #close

### CODE REVIEW

File: docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst [hunk 1]
```diff
@@ -110,59 +110,85 @@ Rebuild The Yum Cache
...
-Set the following variables to repositories of your choosing in
-``/etc/puppetlabs/code/environments/production/hieradata/default.yaml``
+   * simp-adapter-foss:  Version appropriate for FOSS Puppet
+   * simp-adapter-pe:   Version appropriate for Puppet Enterprise
 
-.. code-block:: yaml
+2. Install the simp-adapter package
 
-   # Full URL to a YUM repo for Operating System packages
-   simp::yum::os_update_url: 'http://mirror.centos.org/centos/$releasever/os/$basearch/'
```
Comment (jeefberkey) [inline docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst]: Can we genericize these URLs? I think examples being here makes this guide much more useful.

File: docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst [hunk 2]
```diff
@@ -110,59 +110,85 @@ Rebuild The Yum Cache
...
------------
+   $ sudo yum install -y simp-adapter-foss
 
-Run simp config:
+3. Install the remaining SIMP packages
 
 .. code-block:: bash
 
-   $ simp config
+   $ sudo yum install -y simp
```
Comment (jeefberkey) [inline docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst]: This is a good place to mention simp-extra or whatever it is, as well as the fact that not all modules are installed by default.

They exist under the 'pupmod-*' RPM namespace.

File: docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst [hunk 3]
```diff
@@ -110,59 +110,85 @@ Rebuild The Yum Cache
...
 .. code-block:: bash
 
-   $ simp bootstrap
+   $ reboot
+
 
-Clients
--------
+Bootstrap SIMP Clients
+----------------------
```
Comment (jeefberkey) [inline docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst]: The updated client management guide should suffice for this section.

**Comment by jeefberkey** [comment]
I also think that 'Installation Miscellany' is no longer a good title for that page. Maybe something with `simp config` in the name?

File: docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst (hunk 1 above)
Comment (lnemsick-simp) [inline docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst]: simp config prompts for these URLs.  I'm not sure how you want to document this.

File: docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst (hunk 2 above)
Comment (trevor-vaughan) [inline docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst]: Done in coming update

**Pull request merged**